rezerwacje-app/src/main/java/org/niewidoczniakademicy/rezerwacje/model/database/ExamTerm.java

```diff
@@ -45,6 +49,10 @@
     @NonNull
     private LocalTime timeEnd;
 
+    @NonNull
+    @ColumnDefault("false")
+    private Boolean isDeleted;
```

Comment:
Chyba lepiej nazywać booleany bez _is_, bo później mogą być problemy przy parsowaniu np. Jacksonem:
https://stackoverflow.com/questions/32270422/jackson-renames-primitive-boolean-field-by-removing-is